src/messages/MOSDOpReply.h

```diff
@@ -47,6 +48,8 @@ class MOSDOpReply final : public Message {
   int32_t retry_attempt = -1;
   bool do_redirect;
   request_redirect_t redirect;
+  dmc::PhaseType qos_phase;
+  uint64_t qos_cost;
```

Comment:
uint64_t qos_cost = 1u; here rather than in the default constructor below, easier to audit.

Reply:
Done.